app/views/shared/_form_photocopy.html.erb

```diff
@@ -0,0 +1,10 @@
+<div class="form-group">
+  <% if entity.present? and entity.send(attribute).present? %>
```

Comment:
@jasonoverby You should avoid using `and`, and just always use `&&`. The former can have strange, hard-to-debug consequences due to differences in operator precedence.

Check this out: http://stackoverflow.com/questions/1426826/difference-between-and-and-in-ruby